### ISSUE TITLE
system-wide installation: setting up per-user customization is harder than it needs to be

### ISSUE THREAD

**Issue by manuelschiller** [FemtoEmacs/Femto-Emacs#15]
Please look at the following as a (hopefully respectful) suggestion - Femto-Emacs is beautiful as is!
While Femto-Emacs' per-user customization file(s) are beautifully minimal, using Femto-Emacs on a system with a system-wide installation is essentially a thing for expert users. They have to know where to copy which files (init.lsp and r5rs.scm at the very least) from some system-wide location to their respective home directories.
Having some kind of search path (which points to the user's Femto-Emacs area first, then, if the files can't be found there, picks up any system-wide standard files) would be much more user-friendly (and more in line with how most other editors under *nix-like systems do things). Users who want to customize Femto-Emacs can, and the rest just gets the default startup files. I have hacked together something along those lines (with my very rusty LISP/Scheme from 20 years ago) that seems to work for me, on my system, see https://github.com/manuelschiller/Femto-Emacs - maybe it can be an inspiration (since I'm sure you people can come up with something much nicer that will work on other systems, too).
And thanks for Femto-Emacs - finally a version of emacs I can enjoy (almost irrespective of the CPU power of the system, or rather lack thereof)...

**Comment by hughbarney** [comment]
FemtoEmacs is no longer maintained.

I am the original author of the atto and femto Editors upon which FemtoEmacs is based.
Femto has now caught up to the same level of functionality as FemtoEmacs and is less buggy and more maintainable.

Long term I plan to fix bugs and future enhancements at:
https://github.com/hughbarney/femto

A similar issue is also logged at:
https://github.com/hughbarney/femto/issues/8

I looked at your repo but could not find a list of the hacks and enhancements that you have made.

**Comment by manuelschiller** [comment]
Fair enough. I got very excited when I found Femto-Emacs because I love tiny but useful text editors, and got to hack a bit, and wanted to share the bits that I considered to be potentially useful for a wider audience.
If you still want to look at what I did, you can find a diff for the couple of small commits here: https://github.com/FemtoEmacs/Femto-Emacs/compare/master...manuelschiller:Femto-Emacs:master
Feel free to pick up bits and pieces or reject as you see fit (and close this issue, if you feel like it). ;)
In any case, I'll have a look at femto (and potentially atto). Thanks!

**Comment by hughbarney** [comment]
I have had a look at what you have done.  Most of your changes relate to FemtoLisp setup.
FemtoLisp in the master project 'Femto' at:
https://github.com/hughbarney/femto
